altair/utils/data.py

```diff
@@ -3,12 +3,15 @@
 import random
 import hashlib
 import warnings
+from importlib.metadata import version as importlib_version
+from importlib.metadata import PackageNotFoundError
```

Comment:
nit: These could be consolidated to a single import line

```python
from importlib.metadata import version as importlib_version, PackageNotFoundError
```

Reply:
Was not aware that this is a valid syntax if an import is renamed. Thanks! Done in 81f1146fb36f